### PR TITLE
ci(release): conventional commit

### PR DESCRIPTION
**Pull request by pdesoyres-cc**
Fixes: #330

**We must merge this just after a manual old-school release**

### CODE REVIEW

**Comment by hsablonniere** [comment]
@pdesoyres-cc Looks good to me, can we add some guidance to help contributors chose a scope. Most of the time, it will the component name, but what happens if it's a lib or multiple components? Maybe some examples for some situations.

Also, I was wondering... I think we should handle the next stable release (the one with all the current pending changes) manually as it's still mostly done with our manual Changelog updates, WDYT?

**Comment by pdesoyres-cc** [comment]
I have added some guidance for the commit message.

The next release will be done manually as usual. Just after this release, we must merge this PR and ask everybody to rebase on it and install the githook.

**Review by hsablonniere — APPROVED** [review]

File: docs/guides/how-to-release-the-components.md [hunk 1]
```diff
@@ -3,45 +3,23 @@ kind: 📌 Docs
...
 
-1. Make sure you updated the `CHANGELOG.md` with all the new features and bugfixes.
-1. Decide if you're doing a `major`, `minor` or `patch` version or even a `prerelease` according to the semver spec.
-1. Make sure the commit you want to version is on `master`.
-1. Run the `npm version` command with the right parameters (see details below).
-  * This will update `package.json` and `package-lock.json` in a new commit.
-  * This will create a git tag.
-1. Push the updated `master` branch with the new tag on GitHub (see details below).
-1. Run `npm publish` to deploy the new version on npmjs.com (see details below).
+Release is automatized with [release-please](https://github.com/googleapis/release-please).
```
Comment (florian-sanders-cc) [inline docs/guides/how-to-release-the-components.md:8]: "automatized" exists but it's not used a lot, should probably use "automated" instead

**Review by florian-sanders-cc — APPROVED** [review]

**Review by roberttran-cc — APPROVED** [review]
Very nice! :rocket:

**Pull request merged**